docs/AppService/GetActiveServiceConsent/assets/GetActiveServiceConsent_Allow.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+
+skinparam ParticipantPadding 36
+skinparam defaultFontSize 10
+
+participant AppConsumer
```

Comment:
Maybe rename it to ServiceConsumer

Reply:
This case only applies when an application is the service consumer

Reply:
Might go with `MobileConsumer` for consistency with some of the other diagrams